server/user.js

```diff
@@ -0,0 +1,85 @@
+const { TimeLogger } = require("../src/util");
+const { R } = require("redbean-node");
+const { UptimeKumaServer } = require("./uptime-kuma-server");
+const server = UptimeKumaServer.getInstance();
+const io = server.io;
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
+async function sendUserList(socket) {
+    const timeLogger = new TimeLogger();
+
+    let result = [];
+    let list = (await R.findAll("user")).map(({ id, username, active }) => ({
+        id,
+        username,
+        active
+    }));
+
+    io.to(socket.userID).emit("userList", result);
+
+    timeLogger.print("Send User List");
+
+    return list;
+}
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
+async function getUser(userID) {
+    const timeLogger = new TimeLogger();
+
+    const bean = await R.findOne("user", " id = ? ", [ userID ])
+        .then(({ id, username, active }) => ({
+            id,
+            username,
+            active
```

Comment:
Is this intentional?

Reply:
Of course yes! For security reason, I want to keep control of fields returned by the API, for example, I don't want the hashed password to be returned!!

Reply:
In this case, returning just `{ id, username, active }` (instead of `bean`) would be less error-prone

Reply:
Ok let's do it

Reply:
Done, see last commit